guides/rails_guides/markdown/renderer.rb

```diff
@@ -90,6 +90,7 @@ def convert_notes(body)
           end
         end
 
+        # rubocop:disable Lint/EndAlignment
```

Comment:
How do we fix Rubocop rules? The only option is to exclude the file in `.rubocop.yml`, isn't it?